docs/docker.md

```diff
@@ -1,55 +1,98 @@
-
 # Docker
 Armory is intended to be a lightweight python package which standardizes all evaluations
-inside a docker container. Docker images will be pulled as needed when evaluations are 
-ran.
+inside a docker container.
+
+
+## Images
+There are three docker images that are published to dockerhub for every release of 
+the armory framework:
+
+1. `twosixarmory/tf1:<version>` 
+2. `twosixarmory/tf2:<version>` 
+3. `twosixarmory/pytorch:<version>` 
+
+When using `armory launch` or `armory exec` the framework specific arguments will 
+utilize one of these three images. 
 
-However if there are issues downloading the images (e.g. proxy) they can be built 
-from the repo:
+When running `armory run <path/to/config.json>` the image launched will be whatever is 
+specified in the `docker_image` field. This enables users to extend our base images 
+and run evaluations on an image that has all additional requirements for their defense.
+
+
+## Building Images from Source
+When using a released version of armory, docker images will be pulled as needed when 
+evaluations are ran. However if there are issues downloading the images (e.g. proxy) 
+they can be built from the release branch of the repo:
 ```
+git checkout -b r0.5
 bash docker/build.sh <tf1|tf2|pytorch|all>
 ```
 
-NOTE: if you're a developer working on armory from the master branch, you will need to 
-build the dev containers: 
+## Docker Volume Mounts
+When launching an ARMORY instance several host directories will be mounted within the 
+docker container:
+
+| Host Path   | Docker Path   | 
+|:----------: | :-----------: | 
+| os.getcwd() | /workspace    |  
+| ~/armory/datasets | /armory/datasets |
```

Comment:
Should this be ~/.armory/datasets, etc, or <host armorydir>/datasets?

Reply:
So below I note that those are the default values. @davidslater any thoughts on which way to show it? I'm fine either way just looking for another set of eyes

Reply:
Good catch. Updated.